services/bettersocial/user/BetterSocialBlockUserV2.js

```diff
@@ -13,6 +13,91 @@ const Getstream = require('../../../vendor/getstream');
 const RedisBlockHelper = require('../../redis/helper/RedisBlockHelper');
 const BetterSocialScoreBlockUser = require('../score/block-user');
 
+const getstream_unfollow_sequence = async (token, feedOwner, feedTarget, direct = false) => {
+  try {
+    // Main feed following unfollow user anon feed target
+    if (direct) {
+      await Getstream.feed.unfollowMainFeedFollowingDirect(feedOwner, feedTarget);
+    } else {
+      await Getstream.feed.unfollowMainFeedFollowing(token, feedOwner, feedTarget);
+    }
+    // Main feed unfollow user feed target
+    await Getstream.feed.unfollowUser(feedOwner, feedTarget);
+    // Main feed unfollow user exclusive feed target
+    await Getstream.feed.unfollowUserExclusive(feedOwner, feedTarget);
+  } catch (e) {
+    console.log('Error in block user v2 getstream');
+    console.log(e);
+    return {
+      isSuccess: false,
+      message: e?.message || 'Error in block user v2 getstream'
+    };
+  }
+};
+const sign_a_unfollow_sign_b = async (unfollowParams) => {
+  // Destroy follow from sign A to Sign B
+  await UserFollowUserFunction.userBlock(
+    unfollowParams.UserFollowUser,
+    unfollowParams.UserFollowUserHistory,
+    unfollowParams.targetUserId,
+    unfollowParams.selfUserId,
+    {transaction: unfollowParams.t, postId: unfollowParams.postId}
+  );
+  await getstream_unfollow_sequence(
+    unfollowParams.token,
+    unfollowParams.selfUserId,
+    unfollowParams.targetUserId
+  );
+};
+
+const sign_b_unfollow_sign_a = async (unfollowParams) => {
+  // Destroy follow from sign B to sign A
+  try {
+    await UserFollowUserFunction.userBlock(
+      unfollowParams.UserFollowUser,
+      unfollowParams.UserFollowUserHistory,
+      unfollowParams.selfUserId,
+      unfollowParams.targetUserId,
+      {transaction: unfollowParams.t, postId: unfollowParams.postId}
+    );
+    await getstream_unfollow_sequence(
+      unfollowParams.token,
+      unfollowParams.targetUserId,
+      unfollowParams.selfUserId,
+      true
+    );
+  } catch (e) {
+    console.log('Error in block user v2 getstream');
+    console.log(e);
+    return {
+      isSuccess: false,
+      message: e?.message || 'Error in block user v2 getstream'
+    };
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the function `sign_b_unfollow_sign_a` should also have a success return statement for consistency and clarity.

```diff
+    return {
+      isSuccess: true,
+      message: 'Unfollow sequence completed successfully'
+    };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->